### PR TITLE
WIP: Correction of text attribute

### LINKED ISSUE #642: Text for survey date in extract
Detail, but the specification... uhmmm :frowning: Ok Ok i know. Running gag. But we need to change the Text from "Daten der amtlichen Vermessung, Stand {0}." to "Daten der amtlichen Vermessung. Stand der amtlichen Vermessung: {0}." And the corresponding languages of course.

### PR DESCRIPTION
**Pull request by vvmruder**
fixes #642

### CODE REVIEW

**Comment by vvmruder** [comment]
can you add the correct translations @voisardf & @lopo977

**Comment by vvmruder** [comment]
@lopo977 could you please add the Italian translation of the text :)

**Comment by vvmruder** [comment]
Or ask S.Buettler for it? I do not have him here in github as user, or I don't know his correct name.

**Comment by lopo977** [comment]
@vvmruder fixed italian translations

File: pyramid_oereb/standard/pyramid_oereb.yml.mako [hunk 1]
```diff
@@ -328,9 +328,9 @@ pyramid_oereb:
     # This is a multilingual value. In the minimum, the value for the default language has to be defined.
     base_data:
         text:
-          de: Daten der amtlichen Vermessung, Stand {0}.
+          de: Daten der amtlichen Vermessung. Stand der amtlichen Vermessung: {0}.
           fr: Données de la mensuration officielle, état actuel {0}
```
Comment (voisardf) [inline pyramid_oereb/standard/pyramid_oereb.yml.mako]: Données de la mensuration officielle, état de la mensuration officielle

File: pyramid_oereb/standard/pyramid_oereb.yml.mako [hunk 2]
```diff
@@ -328,9 +328,9 @@ pyramid_oereb:
     # This is a multilingual value. In the minimum, the value for the default language has to be defined.
     base_data:
         text:
-          de: Daten der amtlichen Vermessung, Stand {0}.
-          fr: Données de la mensuration officielle, état actuel {0}
-          it: Dati della misurazione ufficiale, stato attuale {0}
+          de: Daten der amtlichen Vermessung. Stand der amtlichen Vermessung: {0}.
+          fr: Données de la mensuration officielle, état de la mensuration officielle: {0}
+          it: Dati della misurazione ufficiale, stato al {0}
```
Comment (vvmruder) [inline pyramid_oereb/standard/pyramid_oereb.yml.mako]: Shouldn't it be something like this:
"Dati della misurazione ufficiale. Stato della misurazione ufficiale: {0}"

But I'am really not an italien speaker :))

Comment (lopo977) [reply pyramid_oereb/standard/pyramid_oereb.yml.mako]: The literal translation you propose is ok. "Dati della misurazione ufficiale. Stato della misurazione ufficiale: {0}." Our business person, proposed another translation, we use in our internal branch: "Dati della misurazione ufficiale, stato al {0}". Feel free to put your translation, that is also correct.

Comment (lopo977) [reply pyramid_oereb/standard/pyramid_oereb.yml.mako]: Thank you for cooperation, @vvmruder!

Comment (vvmruder) [reply pyramid_oereb/standard/pyramid_oereb.yml.mako]: Thanks for clarification

**Pull request merged**